source/view-model.js

```diff
@@ -4,14 +4,69 @@
 
 /*********************** View-Model **********************/
 "use strict";
+function GameInfo(game){
+    this.getFieldCells = function(){
+        return game.getFieldCells();
+    };
+
+    this.getCurrentStatus = function(){
+        let gameStatus = game.currentStatus;
+        let playerName = gameStatus == 'Tie' ? 'It' : game.getCurrentPlayerName();
+        return  playerName + ' is ' + gameStatus;
+    };
+};
+
+let gamePreset = {
+    players: [
+        {type : 'human', name : 'First', symbol : 'X'},
+        {type : 'computer', name : 'Computer', symbol : 'O'}
+    ],
+
+    setPlayer: function(key, value){
+        this.players.forEach(function(item){
+            for(let i in item){
+                if(i == key && item.hasOwnProperty(i)){
+                    item[i] = value;
+                    break;
+                }
+            }
+        });
+        return this.players;
+    },
+
+    getPlayer: function(){
```

Comment:
Please rename this method to `getPlayers` and add two more methods with the same name `getPlayer` that would return specific player by their name or number in the `players` array.